src/Microsoft.Azure.WebJobs.Host.Storage/BlobStorageConcurrencyStatusRepository.cs

```diff
@@ -39,19 +39,19 @@ public BlobStorageConcurrencyStatusRepository(IConfiguration configuration, IHos
 
             try
             {
-                CloudBlobContainer? container = await GetContainerAsync(cancellationToken);
+                BlobContainerClient? container = await GetContainerAsync(cancellationToken);
                 if (container != null)
                 {
-                    CloudBlockBlob blob = container.GetBlockBlobReference(blobPath);
-                    string content = await blob.DownloadTextAsync(cancellationToken);
+                    BlockBlobClient blob = container.GetBlockBlobClient(blobPath);
+                    string? content = (await blob.DownloadContentAsync(cancellationToken)).Value?.Content?.ToString();
```

Comment:
This implementation looks different than https://github.com/Azure/azure-functions-host/blob/8b5973c2c07e4ae1bb2e46de4453a58c41e600ea/src/WebJobs.Script/Scale/BlobStorageConcurrencyStatusRepository.cs#L53. Were there bugs there you are addressing? E.g. the response should never be null, so I don't see why you need the Value?, etc.

Reply:
Updated with the implementation from Host.